### PR TITLE
Update Operator.sol to v0.8

### PR DESCRIPTION
**Pull request by alexroan**

### CODE REVIEW

File: contracts/src/v0.8/dev/Operator.sol [hunk 1]
```diff
@@ -306,7 +304,7 @@ contract Operator is
     public
     validateAuthorizedSenderSetter()
   {
-    TargetsUpdatedAuthorizedSenders(targets, senders, msg.sender);
+    emit TargetsUpdatedAuthorizedSenders(targets, senders, msg.sender);
```
Comment (connorwstein) [inline contracts/src/v0.8/dev/Operator.sol:307]: woah how did this work before?

File: contracts/src/v0.8/dev/ConfirmedOwnerWithProposal.sol [hunk 2]
```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../interfaces/OwnableInterface.sol";
+
+/**
+ * @title The ConfirmedOwner contract
+ * @notice A contract with helpers for basic contract ownership.
+ */
+contract ConfirmedOwnerWithProposal is OwnableInterface {
```
Comment (connorwstein) [inline contracts/src/v0.8/dev/ConfirmedOwnerWithProposal.sol:10]: Just a rename to be more descriptive?

File: contracts/src/v0.8/dev/ConfirmedOwner.sol [hunk 3]
```diff
@@ -1,79 +1,22 @@
...
-  );
-  event OwnershipTransferred(
-    address indexed from,
-    address indexed to
-  );
-
-  constructor(address newOwner) {
-    s_owner = newOwner;
-  }
+contract ConfirmedOwner is ConfirmedOwnerWithProposal {
```
Comment (connorwstein) [inline contracts/src/v0.8/dev/ConfirmedOwner.sol:10]: I'm guessing this is for backwards compatibility with the stuff already using ConfirmedOwner - couldn't we just update all the 0.8 contracts to use ConfirmedOwnerWithProposal (hard cutover to simplify)?

**Review by connorwstein — APPROVED** [review]

File: contracts/src/v0.8/dev/Operator.sol (hunk 1 above)
Comment (alexroan) [reply contracts/src/v0.8/dev/Operator.sol:307]: I think previous Solidity versions allowed it but threw a warning.

File: contracts/src/v0.8/dev/ConfirmedOwnerWithProposal.sol (hunk 2 above)
Comment (alexroan) [reply contracts/src/v0.8/dev/ConfirmedOwnerWithProposal.sol:10]: The difference between `ConfirmedOwner` and `ConfirmedOwnerWithProposal` is that the deployer can set a proposed owner during construction. 

So `ConfirmedOwner` still exists and extends `ConfirmedOwnerWithProposal` without setting a proposal on construction.

File: contracts/src/v0.8/dev/ConfirmedOwner.sol (hunk 3 above)
Comment (alexroan) [reply contracts/src/v0.8/dev/ConfirmedOwner.sol:10]: https://github.com/smartcontractkit/chainlink/pull/4632#discussion_r660654129

**Comment by alexroan** [comment]
Closing this due to out of date requirements.

**Pull request closed without merging**